Cargo.toml

```diff
@@ -22,6 +21,7 @@ serde-xml-rs = "0.2"
 serde_derive = "1.0"
 serde_json = "1.0"
 smallvec = "0.6.2"
+tempfile = "^3.0"
```

Comment:
The `^` is unnecessary. Please remove.

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

> Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.

Interesting. Thanks.